client/http-client/Cargo.toml

```diff
@@ -35,13 +37,11 @@ jsonrpsee-test-utils = { path = "../../test-utils" }
 tokio = { version = "1.16", features = ["net", "rt-multi-thread", "macros"] }
 
 [features]
-default = ["native-tls"]
-native-tls = ["hyper-rustls/native-tokio", "hyper-rustls/ring", "__tls"]
-webpki-tls = ["hyper-rustls/webpki-tokio", "hyper-rustls/ring", "__tls"]
+default = ["tls"]
 
 # Internal feature to indicate whether TLS is enabled.
```

Comment:
Not internal any more then? :)

Reply:
We had two feature flags for tls (native-tls, webpki-tls) before which was public but we don't need them anymore because we only support rustls-platform-verifier and the internal feature flag is not needed anymore

Reply:
ah ok, I guess you were referring to the outdated comment, removed now